playMode.py

```diff
@@ -0,0 +1,252 @@
+    ver_tile_2 = Tile(blue, 128, 256,384,256,"left.jpg")
+    hor_tile_1 = Tile(dark_pink, 256, 128,0,128,"right.jpg")
+    hor_tile_2 = Tile(dark_pink, 256, 128,256,128,"right.jpg")
+    hor_tile_3 = Tile(dark_pink, 256, 128,128,512,"right.jpg")
+    sm_tile_1 = Tile(center, 128, 128,0,0,"cube copy.jpg")
+    sm_tile_2 = Tile(black, 128, 128,384,0,"cube copy.jpg")
+    sm_tile_3 = Tile(black, 128, 128,0,512,"cube copy.jpg")
+    sm_tile_4 = Tile(black, 128, 128,384,512,"cube copy.jpg")
+    tileList = [ver_tile_1,ver_tile_2,hor_tile_1,hor_tile_2,hor_tile_3,sm_tile_1,sm_tile_2,sm_tile_3,sm_tile_4,caocao_block]
+    return tileList
+
+#背水列阵
+def Beishuiliuzhen():
+    caocao_block = Tile(red, 256, 256,128,256,"cube.jpg")
+    ver_tile_1 = Tile(yellow, 128, 256,0,0,"left.jpg")
+    ver_tile_2 = Tile(blue, 128, 256,384,0,"left.jpg")
+    hor_tile_1 = Tile(dark_pink, 256, 128,128,0,"right.jpg")
+    hor_tile_2 = Tile(dark_pink, 256, 128,0,512,"right.jpg")
+    hor_tile_3 = Tile(dark_pink, 256, 128,256,512,"right.jpg")
+    sm_tile_1 = Tile(center, 128, 128,0,384,"cube copy.jpg")
+    sm_tile_2 = Tile(black, 128, 128,384,384,"cube copy.jpg")
+    sm_tile_3 = Tile(black, 128, 128,128,128,"cube copy.jpg")
+    sm_tile_4 = Tile(black, 128, 128,256,128,"cube copy.jpg")
+    tileList = [ver_tile_1,ver_tile_2,hor_tile_1,hor_tile_2,hor_tile_3,sm_tile_1,sm_tile_2,sm_tile_3,sm_tile_4,caocao_block]
+    return tileList
+
+#调兵遣将
+def Diaobingqianjiang():
+    caocao_block = Tile(red, 256, 256,0,0,"cube.jpg")
+    hor_tile_1 = Tile(dark_pink, 256, 128,0,256,"right.jpg")
+    hor_tile_2 = Tile(blue, 256, 128,0,384,"right.jpg")
+    hor_tile_3 = Tile(dark_pink, 256, 128,256,256,"right.jpg")
+    hor_tile_4 = Tile(yellow, 256, 128,256,384,"right.jpg")
+    hor_tile_5 = Tile(grey, 256, 128,128,512,"right.jpg")
+    sm_tile_1 = Tile(center, 128, 128,256,0,"cube copy.jpg")
+    sm_tile_2 = Tile(black, 128, 128,384,0,"cube copy.jpg")
+    sm_tile_3 = Tile(black, 128, 128,256,128,"cube copy.jpg")
+    sm_tile_4 = Tile(black, 128, 128,384,128,"cube copy.jpg")
+    tileList = [hor_tile_1,hor_tile_2,hor_tile_3,hor_tile_4,hor_tile_5,sm_tile_1,sm_tile_2,sm_tile_3,sm_tile_4,caocao_block]
+    return tileList
+
+#逃之夭夭
+def Taozhiyaoyao():
+    caocao_block = Tile(red, 256, 256,128,256,"cube.jpg")
+    ver_tile_1 = Tile(yellow, 128, 256,0,0,"left.jpg")
+    ver_tile_2 = Tile(blue, 128, 256,0,256,"left.jpg")
+    ver_tile_3 = Tile(blue, 128, 256,384,0,"left.jpg")
+    ver_tile_4 = Tile(blue, 128, 256,384,256,"left.jpg")
+    hor_tile_1 = Tile(dark_pink, 256, 128,128,0,"right.jpg")
+    sm_tile_1 = Tile(center, 128, 128,0,512,"cube copy.jpg")
+    sm_tile_2 = Tile(black, 128, 128,384,512,"cube copy.jpg")
+    sm_tile_3 = Tile(black, 128, 128,128,128,"cube copy.jpg")
+    sm_tile_4 = Tile(black, 128, 128,256,128,"cube copy.jpg")
+    tileList = [ver_tile_1,ver_tile_2,ver_tile_3,ver_tile_4,hor_tile_1,sm_tile_1,sm_tile_2,sm_tile_3,sm_tile_4,caocao_block]
+    return tileList
```

Comment:
it will be better to change the variable names

Reply:
you mean for these "ver_tile_1,ver_tile_2,ver_tile_3,ver_tile_4,hor_tile_1,sm_tile_1,sm_tile_2,sm_tile_3,sm_tile_4,caocao_block"?